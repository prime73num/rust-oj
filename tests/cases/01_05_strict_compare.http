###
# Send request:
POST http://127.0.0.1:12345/jobs HTTP/1.1
content-type: application/json

{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { print!(\"Hello World!\"); }","user_id":0}
###
# Got response:
# HTTP 200
# content-length: 421
# content-type: application/json
# date: Sat, 03 Sep 2022 04:02:13 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Compilation Success","time":0},{"id":1,"info":"","memory":0,"result":"Wrong Answer","time":0}],"created_time":"2022-09-03T04:02:14.435Z","id":0,"result":"Wrong Answer","score":0.0,"state":"Finished","submission":{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { print!(\"Hello World!\"); }","user_id":0},"updated_time":"2022-09-03T04:02:14.435Z"}
