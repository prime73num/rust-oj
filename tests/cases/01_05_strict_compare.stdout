{
  "id": 0,
  "created_time": "2022-09-03T04:02:14.435Z",
  "updated_time": "2022-09-03T04:02:14.435Z",
  "submission": {
    "source_code": "fn main() { print!(\"Hello World!\"); }",
    "language": "Rust",
    "user_id": 0,
    "contest_id": 0,
    "problem_id": 0
  },
  "state": "Finished",
  "result": "Wrong Answer",
  "score": 0.0,
  "cases": [
    {
      "id": 0,
      "result": "Compilation Success",
      "time": 0,
      "memory": 0,
      "info": ""
    },
    {
      "id": 1,
      "result": "Wrong Answer",
      "time": 0,
      "memory": 0,
      "info": ""
    }
  ]
}
